### PR TITLE
First Contribution

### PR DESCRIPTION
**Pull request by SakshiShah29**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4231"></a>

### CODE REVIEW

File: data/Sakshi.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "name":"Sakshi Shah",
+    "type":"personal",
+    "bio":"Open Source Enthusiast!",
+    "avatar":"https://github.com/SakshiShah29.png",
```
Comment (Pradumnasaraf) [inline data/Sakshi.json:5]:
```suggestion
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

In the recent update, we don't need to add `avatar:` anymore, it will auto-pick your GitHub profile picture. So, please remove that.
Updated docs: https://linkfree.eddiehub.io/docs/quickstart

Also, fix your JSON profile file naming. It should be `{your GitHub username}.json` ->  `SakshiShah29.json`. Otherwise, some features will not be functional in the future.

If you need any help feel free to ask.

File: data/Sakshi.json (hunk 1 above)
Comment (SakshiShah29) [reply data/Sakshi.json]: Changes made

File: data/SakshiShah29.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        }
+    ],
+    "milestones":[
+        {
+            "title":"Started Freelancing",
+            "date":"January 2023",
+            "icon":"dollar",
+            "color":"grey",
+            "description":"Started Freelancing",
+            "url":"https://www.eddiejaoude.io/"
```
Comment (kumarsonsoff3) [inline data/SakshiShah29.json:24]: Please update it with your URL instead, or else you can remove this field!

**Review by kumarsonsoff3 — DISMISSED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**